gensim/models/word2vec.py

```diff
@@ -1539,9 +1539,13 @@ def n_similarity(self, ws1, ws2):
           True
 
         """
+        if not(len(ws1) and len(ws2)):
+            raise ZeroDivisionError('Atleast one of the passed list is empty.')
         v1 = [self[word] for word in ws1]
         v2 = [self[word] for word in ws2]
-        return dot(matutils.unitvec(array(v1).mean(axis=0)), matutils.unitvec(array(v2).mean(axis=0)))
+        return dot(matutils.unitvec(array(v1).mean(axis=0)),
```

Comment:
We don't use vertical indent in gensim -- change to normal hanging indent (see PEP8: all statements on new lines, one level of indent).